### PR TITLE
Relax tqdm dependency

### PR DESCRIPTION
**Pull request by feng-tao**
Try to update some deps in etl and found this dependency is pinned and caused failure. I don't think we need to pin this dep and it is used in backfill progress bar.

### CODE REVIEW

**Comment by feng-tao** [comment]
👀  @jinhyukchang @astahlman @ArgentFalcon 
cc @lyft/dp-tools-viz

File: setup.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ def do_setup():
             'sqlalchemy>=1.1.15, <=1.3.0',
             'tabulate>=0.7.5, <0.8.0',
             'thrift>=0.9.2',
-            'tqdm==4.23.4',
+            'tqdm>=4.23.4',
```
Comment (astahlman) [inline setup.py]:
```suggestion
            'tqdm>=4.23.4, <5',
```

**Comment by jinhyukchang** [comment]
👍

**Comment by astahlman** [comment]
👍

**Pull request merged**